Objects/Extracting information from objects.js

```diff
@@ -0,0 +1,16 @@
+// Write a function that takes an object as argument containing properties with personal information
+// Extract firstName, lastName, size, and weight if available
+// If size or weight is given transform the value to a string
+// Attach the unit cm to the size
+// Attach the unit kg to the weight
+// Return a new object with all available properties that we are interested in
+
+function myFunction(obj) {
+  const { fn, ln, size, weight, ...rest } = obj;
+  const obj2 = { fn, ln };
+  if (size) obj2.size = `${size}cm`;
```

Comment:
```suggestion
  if (size) {
    obj2.size = `${size}cm`
  };
```

use `curly braces` wherever applicable. It makes code more readable